src/com/futurewei/alcor/controller/db/ignite/IgniteConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package com.futurewei.alcor.controller.db.ignite;
+
+import com.futurewei.alcor.controller.logging.Logger;
+import com.futurewei.alcor.controller.logging.LoggerFactory;
+import org.apache.ignite.Ignition;
+import org.apache.ignite.client.ClientException;
+import org.apache.ignite.client.IgniteClient;
+import org.apache.ignite.configuration.ClientConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.util.Assert;
+import java.util.logging.Level;
+
+@Configuration
+@ComponentScan("com.futurewei.alcor.controller.cache")
+@EntityScan("com.futurewei.alcor.controller.cache")
+@ConditionalOnProperty(prefix = "ignite", name = "host")
+public class IgniteConfiguration {
+    private static final Logger logger = LoggerFactory.getLogger();
+
+    @Value("${ignite.host}")
```

Comment:
Consider to add SSL support.

Reply:
Yes, but SSL handshake failed in thin client mode, while SSL handshake is normal in non-thin client mode. Besides SSL, I also thinking about transaction too.